src/conf_mode/protocols_bgp.py

```diff
@@ -450,15 +450,15 @@ def verify(bgp):
                             verify_route_map(afi_config['route_map'][tmp], bgp)
 
                 if 'route_reflector_client' in afi_config:
-                    if 'remote_as' in peer_config and peer_config['remote_as'] != 'internal' and peer_config['remote_as'] != bgp['system_as']:
+                    peer_group_as = peer_config.get('remote_as')
```

Comment:
Variable name is missleading - it should be `peer_remote_as` as this is related to the peer configuration itself, and not when using a peer-group but can be changed in a subsequent commit